app/Http/Controllers/FaqController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Faq;
+use Illuminate\Http\Request;
+
+class FaqController extends Controller
+{
+    /**
+     * Display a list of all the faqs
+     */
+    public function index()
+    {
+        return view('faq.index', [
+            'faq' => Faq::all()
+        ]);
+    }
+
+    /**
+     * create faq
+     */
+    public function create()
+    {
+        return view('faq.create');
+    }
+
+    public function store(Request $request)
```

Comment:
Don't forget to add doc blocks to all methods. Make sure you can see these PHPCs comments. This can be viewed in the "Files changed" section of the pull request in GitHub or by running PHPCs on your local machine. See the class materials for that.